server/src/test/java/io/druid/client/CachingClusteredClientFunctionalityTest.java

```diff
@@ -81,7 +82,8 @@ public void testUncoveredInterval() throws Exception {
         .dataSource("test")
         .intervals("2015-01-02/2015-01-03")
         .granularity("day")
-        .aggregators(Arrays.<AggregatorFactory>asList(new CountAggregatorFactory("rows")));
+        .aggregators(Arrays.<AggregatorFactory>asList(new CountAggregatorFactory("rows")))
+        .context(ImmutableMap.<String, Object>of("uncoveredIntervalsLimit", 100));
```

Comment:
are we hitting this limit in the test ? If not can you add a test where we actually hit the limit.